bundles/org.openhab.binding.tradfri/README.md

```diff
@@ -79,6 +79,7 @@ The control outlet supports the `power` channel.
 A blind or curtain supports, beside `battery_level` and `battery_low` channels,  a `positon` channel.
 
 An air purifier supports:
+
```

Comment:
Should the list below also be changed to "- "?

Reply:
You made me go through many files ;-) i adapted as many as i could find.